test/framework/machinedeployment_helpers.go

```diff
@@ -260,40 +264,6 @@ type WaitForMachineDeploymentRollingUpgradeToStartInput struct {
 	MachineDeployment *clusterv1.MachineDeployment
 }
```

Comment:
I think now we can drop `WaitForMachineDeploymentRollingUpgradeToStartInput`. It is unused.

Reply:
Thx, yup!